src/app/admin/sites/[siteSlug]/categories/[categorySlug]/edit/page.tsx

```
@@ -0,0 +1,106 @@
+import React from 'react';
+import Link from 'next/link';
+
+// Try different import methods to increase reliability
+let CategoryForm;
+
+try {
+  const categoryComponents = require('@/components/admin/categories');
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)